install/linux/usr/share/odemis/sim/enzel-sim.odm.yaml

```diff
@@ -253,9 +254,6 @@ ENZEL-Sim: {
   metadata: {
     FAV_POS_ACTIVE: { 'z': 0.002 }
   },
-  persistent: {
-    metadata: [ FAV_POS_ACTIVE ],  # Store "good focus" position between run
-  },
```

Comment:
@Kleijwegt , this part is inconvenient for the test cases, because the image is not necessarily in focus (and it's hard to know the right focus position, if a bad one is stored). I looked the GUI code, and it doesn't seemed to be used anyway (only FAV_POS_ACTIVE on stage and align are used). Could you confirm that's the case?